driver-core/src/test/unit/com/mongodb/internal/connection/ZstdCompressorTest.java

```diff
@@ -0,0 +1,22 @@
+package com.mongodb.internal.connection;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.github.luben.zstd.util.Native;
+
+import java.lang.reflect.Field;
+
+import org.junit.jupiter.api.Test;
+
+final class ZstdCompressorTest {
+    @Test
+    void zstdClassInitializer() throws NoSuchFieldException, IllegalAccessException {
```

Comment:
This test is quite hacky but I think it gives some confidence. But I am also okay with leaving it out.